src/commands/firestore-indexes-list.ts

```diff
@@ -13,13 +13,18 @@ export const command = new Command("firestore:indexes")
     "Pretty print. When not specified the indexes are printed in the " +
       "JSON specification format."
   )
+  .option(
+    "--database <databaseId>",
+    "Database ID of the firestore database from which to list indexes. (default) if none provided."
+  )
   .before(requirePermissions, ["datastore.indexes.list"])
   .before(warnEmulatorNotSupported, Emulators.FIRESTORE)
   .action(async (options: any) => {
     const indexApi = new fsi.FirestoreIndexes();
 
-    const indexes = await indexApi.listIndexes(options.project);
-    const fieldOverrides = await indexApi.listFieldOverrides(options.project);
+    const databaseId = options.database || "(default)";
```

Comment:
Nit: prefer ?? over || here since we only want to fall back to "(default)" if options.database is undefined